sentry/src/test/java/io/sentry/SentryEnvelopeItemTest.kt

```diff
@@ -20,7 +41,138 @@ class SentryEnvelopeItemTest {
         }
     }
 
+    @Test
+    fun `fromAttachment with bytes`() {
```

Comment:
H: I'd add a test specifically for non ascii file content like a binary or so, to be sure this won't break the parsing of the envelope either on serialization nor deserialization.
this file could exist in the `test/resources` folder, maybe just download a random exported `sqlite` database from the webz?